FeedStoreChallenge/Infrastructure/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Avelino Rodrigues on 04/08/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal final class ManagedCache: NSManagedObject {
```

Comment:
`internal` is a default access level, no need to declare it explicitly. Please remove it.

```suggestion
final class ManagedCache: NSManagedObject {
```

Reply:
Thank you @danillahtin, I've just committed this change.